src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -151,11 +136,36 @@ public ColumnValues getFieldValues(UUID entityTypeId, String fieldName, @Nullabl
       return getFieldValuesFromApi(field, searchText);
     }
 
-    if (field.getName().equals("pol_currency")) {
-      return getCurrencyValues();
+    throw new InvalidEntityTypeDefinitionException("Unable to retrieve column values for " + fieldName, entityType);
+  }
+
+  private ColumnValues getSourceTenantIds(EntityType entityType) {
```

Comment:
This method and `getTenantIds()` are both just what we already had, but split out into new methods